migration/sql-files/052-edit-username.sql

```diff
@@ -0,0 +1,2 @@
+-- default is 'false', works with business logic as well.
+ALTER TABLE identities ADD COLUMN username_updated BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
Add a migration test in migration_blackbox_test.go for migration 52 where you check the creation of this column in `identities` table and the value == `false` for the test entities.